packages/form-field/src/FormField/FormField.tsx

```diff
@@ -66,6 +66,7 @@ export const FormField = forwardRef<HTMLDivElement, FormFieldProps>(
               {label && (
                 <Label
                   data-testid="lg-form_field-label"
+                  data-lgid="lg-form_field-label"
```

Comment:
I think i might remove the test id from here and add it to the `label` component. This way i don't have to keep adding it to each component